client/blocks/image-editor/image-editor-canvas.jsx

```diff
@@ -157,34 +156,6 @@ export class ImageEditorCanvas extends Component {
 		this.updateCanvasPosition();
 	}
 
-	toBlob( callback ) {
```

Comment:
Isn't this function which called in https://github.com/Automattic/wp-calypso/blob/trunk/client/blocks/image-editor/index.jsx#L164?

Reply:
Hmm, so that's why I missed it. Disregard, I'm closing this one!